src/imgtools/coretypes/box.py

```
@@ -0,0 +1,385 @@
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+
+import SimpleITK as sitk
+
+from imgtools.logging import logger
```

Ruff (0.8.2)

`imgtools.logging.logger` imported but unused

Remove unused import: `imgtools.logging.logger`

(F401)